src/Libraries/CoreNodeModels/Input/BasicInteractive.cs

```diff
@@ -61,7 +61,6 @@ public override NodeInputData InputData
 
         protected BasicInteractive(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
         {
-            Type type = typeof(T);
```

Comment:
Do we want to simply remove this function?

Reply:
Will check that.

Reply:
It actually has references so I cannot remove it.